lib/core/config.js

```diff
@@ -368,10 +374,33 @@ Config.prototype.loadEmbarkConfigFile = function() {
 };
 
 Config.prototype.loadPipelineConfigFile = function() {
-  var assets = this.embarkConfig.app;
-  for(var targetFile in assets) {
-    this.assetFiles[targetFile] = this.loadFiles(assets[targetFile]);
+
+  const defaultPipelineConfig = {
+    typescript: false
+  };
+
+  const pipelineConfigPath = fs.dappPath(
+    require(fs.dappPath(EMBARK_CONFIG_FILE)).config || DEFAULT_CONFIG_PATH,
```

Comment:
This line is failing inside `test_apps/contracts_app`, when running tests, because `fs.dappPath(EMBARK_CONFIG_FILE)).config` is an object as opposed to a path to the config folder inside the dapp.

The reason this is failing on this branch but not in #960 is because #960 has this logic in side `WebpackProcess` which is not actually launched when `npm run embark test` inside `contracts_app` is executed.

I'm not fully aware why and in which scenarios we expect `embarConfig.config` to **not** be a path (or the other way around), but I'll look into how to work around that.

Reply:
After further research and discussion with the team I've learned that "normal" dapps usually configure just a path for Embark to find config files, while "simple" dapps enable control over single config files, in which case they are defined as part of `embarkConfig.config`.

Updated the PR to address this.